python/py-gwpy/Portfile

```diff
@@ -22,12 +22,12 @@ homepage            https://gwpy.github.io
 master_sites        pypi:g/gwpy \
                     https://github.com/gwpy/gwpy/releases/v${version}/
 
-checksums           rmd160  a463577efc52e969a1f8bfc28190c8d1ff95f651 \
-                    sha256  fd05cfba0a720c11a19694268b42da5f36ca52b7e77d605946fbbecc7ca5b162 \
-                    size    1210901
+checksums           rmd160 bccabf53af0def9faeb274aff42ed40250578ba5 \
+                    sha256 74c5066e19e2b10dd0571ac0f0bd588b302d1f6de639a994021a580bb90d93f4 \
+                    size   1205640
 
 python.versions     27 36 37
-python.default_version 27
+python.default_version 37
```

Comment:
I think this line should probably just be removed entirely

Reply:
@mf2k, why is that?

Reply:
The python portgroup has the line already. I think it is best to just use that default `default_version`.

Reply:
thanks @mf2k, I have made the requested changes.